src/data/fin/interval.lean

```diff
@@ -152,4 +157,80 @@ by rw [fintype.card_of_finset, card_Iic]
 by rw [fintype.card_of_finset, card_Iio]
 
 end unbounded
+
+section filter
+
+variable {n}
```

Comment:
You're using them quite a lot.
```suggestion
variables {n} (a b : fin n)
```
Also, you can shorten all the `finset.univ` to `univ`. We're in finset-land 😜